src/components/Web3Name/Web3Name.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styles from './Web3Name.module.css'
+
+const W3N_URL = process.env.W3N_URL || 'https://w3n.id/'
+const W3N_PREFIX = process.env.W3N_PREFIX || 'w3n:'
+
+export interface Props {
+  name: string
+}
+
+export const Web3Name: React.FC<Props> = ({ name }) => {
+  return (
+    <>
```

Comment:
There’s no need for `<>` here and for `</>` below

Reply:
fixed in [52ef83b](https://github.com/BTE-Trusted-Entity/stakeboard/pull/68/commits/52ef83b92c06e968c819e7e3bd4d01d485290b0e)